website/docs/sql_queries.md

```diff
@@ -337,10 +326,8 @@ will be supported in the future.
 
 ## StarRocks
 
-Copy on Write tables in Apache Hudi 0.10.0 and above can be queried via StarRocks external tables from StarRocks version
```

Comment:
The commit message suggests incremental queries are not supported. Can we clarify explicitly if its supported or not ?

Reply:
No, I ran the example mentioned [here](https://github.com/StarRocks/demo/tree/master/documentation-samples/hudi) incremental queries are not supported in both MOR and COW.